Sources/ISDBTibs/TibsBuilder.swift

```diff
@@ -276,23 +352,36 @@ extension TibsBuilder {
   }
 
   public func writeNinjaRules<Output: TextOutputStream>(to stream: inout Output) {
+#if os(Windows)
+    let callCmd = "cmd.exe /C "
+#else
+    let callCmd = ""
+#endif
+    let swiftIndexCommand = callCmd + """
+\(escapeCommand([toolchain.swiftc.path])) $in $IMPORT_PATHS -module-name $MODULE_NAME \
```

Comment:
Indentation

Reply:
Oh neat, I didn't realize that indentation in multilined strings looked at the indent of the first line.